reposense/RepoSense
# Use header files (not tags) to show/hide the version alert

For details of this PR, see the proposed commit message.

Testing instructions:
* See the alert appearing at the top of UG pages. A similar alert will appear in the DG pages of the production site.
* See the topNav for the extra link to the production website. This link will not appear in the production website (appears only in the dev website).
* See the `DG > Workflow > Writing documentation` section

Proposed commit message:
```
Use header files (not tags) to show/hide the version alert

The 'version alert' that tells readers whether they are looking at
the production or the dev website is an <include> in every
DG page. Showing/hiding it is controlled by the MarkBind
tags mechanism.

Let's use the header mechanism instead, which does not require any
changes to the DG pages as it is controlled by the site.json entirely.
At the same time, let's also add an alert in the UG of the dev website
to alert readers in case a user accidentally ended up in the dev
website when intending to go to the production website.

The above change removes the use of tagging in the website entirely.
However, the tagging mechanism might be useful for other variations
between the two sites. Let's include an extra link in the topNav of
the dev website to point to the production website, as an example of
content controlled by tags. Let's also update the DG to explain how
to use the tagging mechanism to control contents.
```
